src/lib.rs

```diff
@@ -147,6 +170,24 @@ pub enum SnmpError {
 
 pub type SnmpResult<T> = Result<T, SnmpError>;
 
+pub struct ResponseItem {
+    address: String, // like IPv4('192.168.1.1') or IPv6(_)
```

Comment:
I think it's better to keep the original `SocketAddr` here without conversion to String.

Reply:
I can use IpAddr here, because we do not need the port here. Port will be the same for all responses.

Reply:
Yes it's ok